src/guide/1/geofencing/javascript/snippets/createdoc.test.yml

```diff
@@ -0,0 +1,14 @@
+---
+name: index#createdoc
```

Comment:
We already give that prefix (`index`) to other snippet tests try something else such as `guide-geofencing#create`.
This review applies for snippet file name too.